README.md

```diff
@@ -49,7 +49,7 @@ You can see a (customized) instance of Strapping in action by visiting http://oV
 2. Clone the repository:
 
    ```
-   git clone https://github.com/OSAS/strapping-mediawiki strapping
+   git clone https://github.com/axxie/strapping-mediawiki strapping
```

Comment:
Uh, I see.
Looks like original author, @garrett, abandoned the project long time ago.